db/schema.rb

```diff
@@ -10,7 +10,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2022_04_29_144309) do
```

Comment:
Can you run `rake db:migrate` locally and commit the changes? I don't think this change is relevant to the PR.

Reply:
No it shouldnt be.. i did do it - not sure why its pushing up that file. Can we not remove it from the PR @jamesrose